mailbox/plugin/deleted-messages-vault/src/main/java/org/apache/james/vault/DeletedMessageConverter.java

```diff
@@ -61,10 +61,15 @@ DeletedMessage convert(DeletedMessageVaultHook.DeletedMessageMailboxContext dele
             .sender(retrieveSender(mimeMessage))
             .recipients(retrieveRecipients(mimeMessage))
             .hasAttachment(!message.getAttachments().isEmpty())
+            .size(messageSize(message))
             .subject(mimeMessage.map(Message::getSubject))
             .build();
     }
 
+    private long messageSize(org.apache.james.mailbox.store.mail.model.Message message) {
+        return message.getBodyOctets() + message.getHeaderOctets();
```

Comment:
I still think it should be a method of Message

Reply:
there is a method `getFullContentOctets()` already exist. I checked its implementations and see they do the same business, returns a sum of body octests + header octests.

But there is a warning from @chibenwa for using `getFullContentOctets()` regards to some technical issues.

Reply:
We can't duplicate a behavior in other places because the one we would like to call has some issues. We need to fix the issue otherwise we'll have a very bad software in the end.
@chibenwa : can you give us links to problems you mentioned?